Teensy/src/gpio/gpio.cpp

```diff
@@ -94,37 +95,22 @@ static GyroData_s gpio_getGyro()
     return gyro;
 }
 
+// defining resistors for voltage divider
+constexpr float BATTERY_SCALER = (4000.0f + 1000.0f ) / 1000.0f;
```

Comment:
Sorry, this is my fault but just realized I was using the wrong spelling of scaler. It should be scalar